lib/src/repo_path.rs

```diff
@@ -317,6 +317,7 @@ mod tests {
     fn parse_fs_path_wc_in_cwd() {
         let temp_dir = testutils::new_temp_dir();
         let cwd_path = temp_dir.path().join("repo");
+        #[allow(clippy::redundant_clone)] // Workaround nightly clippy false positive
```

Comment:
It's not a false positive because we can `let wc_path = &cwd_path`.

OTOH, I think it's okay to add `#[allow(clippy::redundant_clone)]` to some test functions. Removing `.clone()` from the last use of `id` is waste of effort, and it makes harder to add more assertions.

Reply:
That's nice, thanks! I'll update this at some point in the near future.

Unless I misread clippy's message, it suggested that just removing `.clone()` is OK here. So, I'd still call it a false positive. I'm not sure if I'll get to turning this into a minimal example and filing it as a bug.

Reply:
You got hit by https://github.com/rust-lang/rust-clippy/issues/10577.